build_product

```diff
@@ -324,11 +339,12 @@ done
 
 CMAKE_OPTIONS=(${ADDITIONAL_CMAKE_OPTIONS} "${build_type_option}" "${oval_major_version_option}" "${oval_minor_version_option}" '-DSSG_PRODUCT_DEFAULT=OFF' "${cmake_enable_args[@]}" -G "$cmake_generator")
 set_no_derivatives_options
+set_quick_options
 
 set -e
 rm -rf build/*
 cd build
 cmake .. "${CMAKE_OPTIONS[@]}"
-$build_command "-j${jobs}"
+$build_command "-j${jobs}" ${QUICK}
```

Comment:
With an array, you could use `"${QUICK[@]}"`, which is generally considered better, as it can handle whitespaces.